src/controllers/InMemoryTaskManager.java

```diff
@@ -3,7 +3,6 @@
 import java.util.*;
 
 import model.*;
-import org.junit.jupiter.api.extension.ExtendWith;
 
 public class InMemoryTaskManager implements ITaskManager {
     private  HistoryManager historyManager = null;
```

Comment:
Поле должно быть final
```suggestion
    private final HistoryManager historyManager;
```